ui/src/app/controller/services/mapping/timestamp-mapping.service.ts

```diff
@@ -125,6 +125,19 @@ export class TimestampMappingService {
     }
   }
 
+  HideTerminalOptions(timestampType: PortcallTimestampType): boolean {
+    console.log(timestampType);
+    switch (timestampType) {
+      case PortcallTimestampType.ETA_PBP:
+      case PortcallTimestampType.RTA_PBP:
+      case PortcallTimestampType.PTA_PBP:
+      case PortcallTimestampType.ATA_PBP:
```

Comment:
This is missing a couple of JIT1.1 timestamps too such as:

  * SOSP
  * EOSP
  * ATC-Pilotage
  
Not sure if the omission of JIT 1.1 was intentional (it can be just you keep them in mind for when you do JIT1.1)